pkg/cmd/openshift-sdn-node/proxy.go

```diff
@@ -135,6 +136,10 @@ func (d *sdnLocalDetector) IfNotLocal() []string {
 	return []string{"!", "-i", sdnnode.Tun0}
 }
 
+// func (d *sdnLocalDetector) IfLocalNFT() []string {
+//	return []string{}
+//}
```

Comment:
yeah, you'll need to add `IfLocalNFT` and `IfNotLocalNFT` to obey the interface but they can just return `nil` because they'll never get called